docs/it-manual/configure-gis-service.md

```diff
@@ -1,51 +1,149 @@
-# How to Configure your GIS service for address correction and service area validation
+# How to configure your GIS service for address correction and service area validation
 
-There are several configuration values needed to enable address correction and service area validation. These features rely on an external Esri service and are not compatible with other geolocation services.
+You may have some programs that need to collect an applicant's address to offer
+services. Those programs may also want to verify that the address exists and
+matches a standard format, and may also want to verify that the address is in
+the area that's eligible for the program. For example, a program may only be for 
+applicants living in a certain county. CiviForm uses the external Esri service
+for both these use cases. CiviForm is not currently compatible with other
+geolocation services.
 
-The address correction feature may be enabled on its own, but service area validation depends on address correction being enabled.
+## Address Correction
 
-## Configure Address Correction
+Programs may want to have an applicant's address corrected so that they can
+verify the address exists and have all addresses in a standard format. This
+needs to be configured both at the deployment level and at a question level.
 
-[ESRI_ADDRESS_CORRECTION_ENABLED](https://github.com/civiform/civiform/blob/fd0aaa002e2ee01d378ca90f236c316641ed0101/server/conf/application.conf#L725) - Boolean
+If you want to see what results address correction might provide, you can
+experiment with ArcGIS's world endpoint. [Here's a sample query for 700 5th Ave, Seattle WA 98101](https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/findAddressCandidates?Address=700+5th+Ave&City=Seattle&Region=WA&Postal=98101&f=pjson).
```

Comment:
**praise**: Love that this example is here! also seattle has way too many similar addresses that differ only by the cardinal direction that is for some reason _after_ the street name